File: src/main/java/no/jstien/bikeapi/tsdb/read/TimeSerie.java
package no.jstien.bikeapi.tsdb.read;

import java.util.ArrayList;
import java.util.List;

public class TimeSerie {
    private List<DataPoint> values;

    public TimeSerie() {
        this.values = new ArrayList<>();
    }

    public void addDataPoint(DataPoint dataPoint) {
        this.values.add(dataPoint);
    }

    public List<DataPoint> getDataPoints() {
        return values;
    }
}

File: src/main/java/no/jstien/bikeapi/tsdb/read/TimeSerie.kt
package no.jstien.bikeapi.tsdb.read

import java.util.ArrayList

class TimeSerie {
    private val values: MutableList<DataPoint>

    val dataPoints: List<DataPoint>
        get() = values

    init {
        this.values = ArrayList()
    }

    fun addDataPoint(dataPoint: DataPoint) {
        this.values.add(dataPoint)
    }
}
